### PR TITLE
Analytic script

### PR DESCRIPTION
**Pull request by Jaeoc**
Perhaps not the prettiest  script, but works quite well and is readily understandable (I think)

### CODE REVIEW

File: r-script/analysis-marginal.r [hunk 1]
```diff
@@ -9,8 +9,11 @@ dat <- read.csv("test_sample_marginal_dataset.csv", stringsAsFactors = FALSE)
 #Add a variable indicating whether a p-value appears to reported as marginally significant
 dat$marginal <- grepl("marginal|approach", dat$pre) | grepl("marginal|approach", dat$post)
 
+<<<<<<< HEAD
```
Comment (chartgerink) [inline r-script/analysis-marginal.r]: You need to remove the conflicting lines here, line 13 and 16.

File: writing/method-marginal.rmd [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
+
+![**Figure 1.** Flowchart illustrating the data cleaning process](C:/Users/Anton2/Documents/GIT/2016marginal/flowchart_marginal.png)
```
Comment (chartgerink) [inline writing/method-marginal.rmd]: Better to use a code chunk like this 

```
```{r xkcd-1810, fig.cap="_Figure 1._ A comical example of the diversity in communication channels.", out.width="60%", fig.align="center", echo = FALSE}
knitr::include_graphics('figs/xkcd_1810-cc-by-nc-2_5.png', auto_pdf = TRUE)
```"
```
(just remove " from that last line, couldn't get it to show all code immediately(

File: r-script/analysis-marginal.r (hunk 1 above)
Comment (Jaeoc) [inline r-script/analysis-marginal.r]: Hmm, thought, I had.. Tried to cancel a pull request about half-way and do it a different way, which is why things look at bit messy. Fixed the conflict now at least.

File: writing/method-marginal.rmd (hunk 2 above)
Comment (Jaeoc) [inline writing/method-marginal.rmd]: Cool! Didn't realize that was possible with extraneous image files!

**Pull request merged**